template/module/__openerp__.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# © <YEAR(S)> <AUTHOR(S)>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Module name",
+    "summary": "Module summary",
+    "version": "8.0.1.0.0",
+    "category": "Uncategorized",
+    "license": "AGPL-3",
+    "website": "https://odoo-community.org/",
+    "author": "<AUTHOR(S)>, Odoo Community Association (OCA)",
```

Comment:
I a past discussion was remove the space between each author. (I don't have the reference now)